i18n.js

```diff
@@ -0,0 +1,7 @@
+const NextI18Next = require('next-i18next').default;
+
+module.exports = new NextI18Next({
+  browserLanguageDetection: true,
```

Comment:
This is really cool :)

Reply:
I need to test it 'cause i don't think is working as expected right now 🤔